package.json

```diff
@@ -37,9 +36,7 @@
     "gulp-uglify": "1.5.2",
     "gulp-util": "3.0.7",
     "run-sequence": "1.1.5",
-    "swagger-js-codegen": "1.12.0"
-  },
-  "dependencies": {
+    "swagger-js-codegen": "1.12.0",
     "angular": "^1.8.3",
```

Comment:
did you mean to move all of these as well to devDependencies? As you mentioned, no consumer needs these npm dependencies, just the output of this package

Reply:
If we merge this change [every dependency will be a devDependency](https://github.com/OffTheRecordApp/otr_shared_backend_generated_client/blob/9e0a215befd008f9b267c9446e2cb7891fa80e33/package.json#L26)